<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditBanForm</class>
 <widget class="QDialog" name="EditBanForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Danh sách bàn</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QColumnView" name="columnView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>821</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>859</x>
     <y>70</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Thêm bàn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteButton">
   <property name="geometry">
    <rect>
     <x>859</x>
     <y>170</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Xóa bàn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OutButton">
   <property name="geometry">
    <rect>
     <x>859</x>
     <y>510</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Thoát</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ngoaiSanButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ngoài sân</string>
   </property>
  </widget>
  <widget class="QPushButton" name="trongNhaButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Trong nhà</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lau2Button">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>80</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Lầu 2</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>129</y>
     <width>761</width>
     <height>441</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QListWidget" name="ngoaiSanList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>741</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QListWidget" name="trongNhaList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>741</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QListWidget" name="lau2List">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>741</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <zorder>columnView</zorder>
  <zorder>label</zorder>
  <zorder>addButton</zorder>
  <zorder>DeleteButton</zorder>
  <zorder>OutButton</zorder>
  <zorder>ngoaiSanButton</zorder>
  <zorder>trongNhaButton</zorder>
  <zorder>lau2Button</zorder>
  <zorder>stackedWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
